plugins/search/src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -164,18 +173,104 @@ const SelectFilter = ({
   );
 };
 
+const SelectMultipleFilter = ({
```

Comment:
I wonder if we can modify the `<SearchFilter.Select multiple />` to accept an optional _multiple_ prop instead of create a new "SelectMultipleFilter".

We can use the `<Autocomplete />` component from the `@material-ui/lab` package (it is a dependency already):

- Example for single value:
   https://v4.mui.com/components/autocomplete/#combo-box

- Example with multiple values:
   https://v4.mui.com/components/autocomplete/#combo-box

What do you think?